microsoft/testsuites/nvme/nvme.py

```diff
@@ -1,14 +1,45 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 import math
+from typing import cast
 
 from assertpy import assert_that
 
 from lisa import Environment, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
 from lisa.features import Nvme
+from lisa.operating_system import Posix
 from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
 from lisa.testsuite import simple_requirement
-from lisa.tools import Lscpu
+from lisa.tools import Cat, Lscpu, Nvmecli
+from lisa.util import SkippedException
+
+
+def _unmount_disk(node: Node, namespace: str, destory_mountpoint: bool = True) -> None:
+    mount_point = namespace.rpartition("/")[-1]
+    node.execute(f"umount {mount_point}", shell=True, sudo=True)
+    if destory_mountpoint:
+        node.execute(f"(echo d; echo w) | fdisk {namespace}", shell=True, sudo=True)
+        node.execute(f"rm -rf {mount_point}", shell=True, sudo=True)
+
+
+def _format_mount_disk(
+    node: Node,
+    namespace: str,
+    file_system: str,
+) -> None:
+    posix_os: Posix = cast(Posix, node.os)
+    posix_os.install_packages("xfsprogs")
```

Comment:
If the package is installed, it shouldn't be installed again. If it's a subclass of Tool, the installed or not can be tracked easily. Please create a tool, instead of calling it like a method. If the tool is used by nvme test cases only, it can be in this file.